### PR TITLE
Parse multi-line template literal interpolations. Fixes #49

### LINKED ISSUE #49: "Unknown word" syntax error for multiline arrow function
When using a multiline arrow function "Unknown word" error is thrown.

I'm using the latest stylelint@9.10.1 which uses postcss-jsx@0.36.0:

```
> stylelint "**/*.styled.js"

Button.styled.js
 8:3  ✖  Unknown word   CssSyntaxError
```

```js
import { css } from 'styled-components';

export const buttonStyles = css`
  border-radius: 4px;

  cursor: pointer;

  ${(props) => {
    return props.small && 'padding: 30px;';
  }}
`;
```

This issue is similar to https://github.com/gucong3000/postcss-styled/issues/37.

### PR DESCRIPTION
**Pull request by hudochenkov**
Fixes #49.

Tested on my main project, where I had problems in three files, and everything good now :)

### CODE REVIEW

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock = false
```
Comment (hudochenkov) [inline .npmrc]: There is no `package-lock.json`. This prevents automatic `package-lock.json` creation.

File: template-parser-helper.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 "use strict";
 const Literal = require("./literal");
-const isLiteral = token => token[0] === "word" && /^\$\{.*\}$/.test(token[1]);
+const isLiteral = token => token[0] === "word" && /^\$\{[\s\S]*\}$/.test(token[1]);
```
Comment (hudochenkov) [inline template-parser-helper.js]: Allow all space and non-space symbols.

File: test/css-in-js.js [hunk 3]
```diff
@@ -141,6 +141,7 @@ describe("CSS in JS", () => {
 	describe("objectify for css", () => {
 		cases.each((name, css) => {
 			if (name === "bom.css") return;
+			if (name === "custom-properties.css") return;
```
Comment (hudochenkov) [inline test/css-in-js.js]: New `postcss-parser-tests` fails on new test file. As it out of scope for this PR, I skip it.

**Comment by hudochenkov** [comment]
Codecov is hanging, because [it's down](https://status.codecov.io/).

**Comment by ai** [comment]
@gucong3000 I will accept PR and release it after 2 days. Stop me if you need some time to resolve it :).

**Comment by ai** [comment]
I merged it manually

**Comment by ai** [comment]
I released it in 0.36.1 to help maintainer since it was critical

**Comment by hudochenkov** [comment]
Thank you!

**Pull request closed without merging**